src/components/JournalEntry/ViewJournalEntry.tsx

```diff
@@ -17,7 +17,7 @@ export const ViewJournalEntry = ({ content }: { content: JSONContent }) => {
   }
 
   return (
-    <div className="m-3 rounded-md border-black border-solid border-2 sm:w-8/12">
+    <div className="m-3 rounded-md border-black border-solid border-2 sm:w-8/12 w-5/6">
```

Comment:
Same as the previous component. A good practice for React component is to NOT set a specific width (unless it's a really fixed size), but instead send the width via a `className` prop from the parent.